src/BaseUrlSender.js

```diff
@@ -6,7 +6,7 @@ class BaseUrlSender {
 
 	send(request) {
 		return new Promise((resolve, reject) => {
-			request.baseUrl = this.urlOverride;
+			request.baseUrl = `${this.urlOverride}${request.baseUrlParams ? `/${request.baseUrlParams}` : ""}`;
```

Comment:
The way we have it formatted right now is that the params is a string. If wanted to support multiple params then it would need to be an array and we would need to know what order to append them in. Do you think we should support for that now or just rename the variable to be singular instead of plural (`baseUrlParam`)

Reply:
I'm thinking probably just rename it to `baseUrlParam`